process/sandbox.md

```diff
@@ -0,0 +1,104 @@
+
+1. That a project has a legally neutral home that is stable and known
+
+2. And that a project *may *attain Incubation-level success faster: 
+
+    1. Due to public visibility and association with the CNCF mission
+
+    2. Through alignment with other CNCF projects (if and only if desired)
+
+    3. Via CNCF Service Desk etc.
+
+3. The CNCF will help projects adopt good principles of governance
+
+### Neutral Home
+
+A neutral home for your project increases the willingness of developers from other companies and independent developers to collaborate, contribute, and become committers. Neutrality requires that projects contribute their trademark to CNCF so that: 
+
+* no company is favored over any other
+* CNCF ensures project governance is transparent and fair for everyone.
+
+### Clarifying Marketing Expectations
+
+To date the CNCF has invested in marketing to educate users and grow awareness of cloud native purpose and benefits, to foster community, and to accelerate production use of projects.  
+
+Investments fall into at least three types: 
+
+* Developer community support: hangouts, meetups, events and (some) conferences
+* Digital marketing: help with online content, interactive tutorials, webinars, and social
+* Product marketing: conference promotion, certification, case studies, AR/PR  
+
+Since the Sandbox is for early stage, sandbox projects will receive minimal marketing support from the foundation.  The Sandbox group as a whole may be promoted from time to time.
+
+The CNCF will lean towards developer community support and the CNCF service desk, to help discovery and initial steps towards CNCF Incubation.  There will only be limited CNCF investment in Digital and Product marketing for individual Sandbox projects, and these should be factual and informative.  
+
+Some key points:
+
+* Sandbox projects will be listed separately from other CNCF projects (cncf.io/sandbox)
+* They will not be prominently listed at our events or issued a press release
+* Reviewed on an annual basis; submit a report to the TOC for review
+* CNCF Sandbox projects can stay in the sandbox indefinitely
+
+### Sandbox Entry Requirements
+
+* Require 2 TOC sponsors to enter the sandbox
+* Require presentation to the TOC community twice-a-month meeting
+* Require adherence to CNCF IP Policy (including trademark transferred)
+* Require sandbox projects to list their sandbox status prominently on website/readme
+
+### Sandbox Exit Requirements
+
+* Existing incubation criteria: [https://github.com/cncf/toc/blob/master/process/graduation_criteria.adoc](https://github.com/cncf/toc/blob/master/process/graduation_criteria.adoc)
+
+    * Require 2/3 vote from the TOC to enter incubation
+    * Used successfully in production by at least three independent end users which, in the TOC’s judgement, are of adequate quality and scope.
+    * Have a healthy number of committers
+
+* Archiving criteria: projects may be archived via a 2/3 vote from the TOC
```

Comment:
I don't see a reference to "archiving" elsewhere, is this intended to mean archive in the github project sense, for experiments I can see that making sense.

For other projects, something like the "In the case of an inception stage project that is not renewed with CNCF, the trademark will be returned to the project maintainers or an organization they designate" would make sense, if development is planned to continue but not in CNCF would also seem a reasonable exit process.

Reply:
I agree w/ @justincormack's statement.

Reply:
@justincormack Archiving means that the project is no longer supported by CNCF (for whatever reason, TOC is empowered to make that decision), similar to how the Apache Attic works or Eclipse Foundation archiving process (we aren't blazing trails here).

One thing we will do in coming weeks is to define the archiving criteria in more detail as we haven't had a project that had to do this yet.

Regarding the trademark case, we would have to do this on a case by case basis and is complicated as it can only go to another foundation (not a for profit entity)

Reply:
Ah ok, that trademark wording is from the existing wording for inception [1] so I guess that should be clarified too...

[1] https://www.cncf.io/projects/graduation-criteria/